bower.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "jquery.overlay",
+  "version": "0.0.4",
+  "homepage": "https://github.com/yuku-t/jquery-overlay",
+  "authors": [
+    "Ouadie BOUSSAID <ouadie.boussaid@gmail.com>"
```

Comment:
yep, was generated by `bower init` you can accept the pull request then changed it..